bodhi/server/models.py

```diff
@@ -735,6 +735,75 @@ def unpush(self, koji):
 
 
 class Update(Base):
+    """
+    This model represents a package.
+
+    This model uses single-table inheritance to allow for different package types.
+
+    Attributes:
+        title (unicode): The update's title which uniquely identifies the update.
```

Comment:
Would it be good to document all the weirdness with this field, like that it is generally an ordered list of the build NVRs contained in the update? This is fine if you want to keep it simple.